ena-submission/src/ena_deposition/ena_submission_helper.py

```diff
@@ -379,57 +329,9 @@ def create_flatfile(
         with tempfile.NamedTemporaryFile(delete=False, suffix=".embl") as temp:
             filename = temp.name
 
-    seqIO_moleculetype = {
-        MoleculeType.GENOMIC_DNA: "DNA",
-        MoleculeType.GENOMIC_RNA: "RNA",
-        MoleculeType.VIRAL_CRNA: "cRNA",
-    }
-
-    embl_content = []
-
-    multi_segment = set(unaligned_nucleotide_sequences.keys()) != {"main"}
-
-    for seq_name, sequence_str in unaligned_nucleotide_sequences.items():
-        if not sequence_str:
-            continue
-        reference = Reference()
-        reference.authors = authors
-        sequence = SeqRecord(
-            Seq(sequence_str),
-            id=f"{accession}_{seq_name}" if multi_segment else accession,
-            annotations={
-                "molecule_type": seqIO_moleculetype[moleculetype],
-                "organism": organism,
-                "topology": organism_metadata.get("topology", "linear"),
-                "references": [reference],
-            },
-            description=description,
-        )
-
-        source_feature = SeqFeature(
-            FeatureLocation(start=0, end=len(sequence.seq)),
-            type="source",
-            qualifiers={
-                "molecule_type": str(moleculetype),
-                "organism": organism,
-                "country": country,
-                "collection_date": collection_date,
-            },
-        )
-        sequence.features.append(source_feature)
-
-        with tempfile.NamedTemporaryFile(delete=False, suffix=".embl") as temp_seq_file:
-            SeqIO.write(sequence, temp_seq_file.name, "embl")
-
-        with open(temp_seq_file.name, encoding="utf-8") as temp_seq_file:
-            embl_content.append(temp_seq_file.read())
-
-    final_content = "\n".join(embl_content)
-
     gzip_filename = filename + ".gz"
```

Comment:
Doesn't this make it `sequences.embl.gz.gz`?

Reply:
The full definition of the filename is:

```python
    if dir:
        os.makedirs(dir, exist_ok=True)
        filename = os.path.join(dir, "sequences.embl")
    else:
        with tempfile.NamedTemporaryFile(delete=False, suffix=".embl") as temp:
            filename = temp.name

    gzip_filename = filename + ".gz"

    Path(gzip_filename).write_bytes(response.content)

    return gzip_filename
```

so it's just one `.gz` - maybe you just didn't see it so easily in the GH change display.